apps/crn-server/test/controllers/research-outputs.test.ts

```diff
@@ -132,12 +132,17 @@ describe('ResearchOutputs controller', () => {
         new GenericError(),
       );
 
-      expect(
+      await expect(
         researchOutputs.create(getResearchOutputCreateData()),
       ).rejects.toThrow(GenericError);
     });
 
     test('Should create the new research output and return it', async () => {
+      const mockDate = new Date('2010-01-01');
```

Comment:
can you use jest timer mocks with modern timers https://jestjs.io/docs/26.x/timer-mocks  ?

Reply:
I did try that but tests started to fail with that "took more than 5 secs to run" error. I will double check it but it seems we'd need to update Jest in order to get it working correctly. Will keep you updated.

Reply:
So I've tried again with no success. It seems that jest 26 doesn't work well with fakerTimers('modern'). Either I get the timeout issue or the controller doesn't use the fake time set